packages/vuetify/src/components/VField/VField.sass

```diff
@@ -98,9 +98,12 @@
     @at-root
       @include tools.density('v-input', $input-density) using ($modifier)
         @at-root #{selector.nest(&, $root)}
-          --v-input-control-height: #{$field-control-underlined-height + $modifier}
```

Comment:
VTextarea uses this to determine the base height with auto-grow

Reply:
🤔

Reply:
Made a new mixin to return static value during compiling